### PR TITLE
bmc/syslog: add implementation

### PR DESCRIPTION
**Pull request by nest1ing**
This brings an implementation of the remote logging settings for all
our platforms.

Signed-off-by: Alexander Filippov <a.filippov@yadro.com>

### CODE REVIEW

File: commands/bmc.nicole [hunk 1]
```diff
@@ -191,11 +195,62 @@ function cmd_ifconfig {
...
+        fi
+        echo
+    fi
+}
+
+# @sudo cmd_syslog_set admin
+# @doc cmd_syslog_set
+# Configure remote syslog server
+#  address[:port] - to change settings
+#  empty - to reset settings
```
Comment (artemsen) [inline commands/bmc.nicole:236]: I would suggest to make a subcommand for resetting, lomething like `bmc syslog reset`.

Comment (nest1ing) [reply commands/bmc.nicole:236]: done

File: commands/bmc.vegman [hunk 2]
```diff
@@ -190,6 +194,71 @@ function cmd_ifconfig {
...
+        fi
+        echo
+    fi
+}
+
+# @sudo cmd_syslog_set admin
+# @doc cmd_syslog_set
+# Configure remote syslog server
+#  address[:port] - to change settings
+#  empty - to reset settings
```
Comment (artemsen) [inline commands/bmc.vegman:236]: Not needed anymore.

Comment (AlexanderAmelkin) [reply commands/bmc.vegman:236]: It still works as far as I can see

**Review by AlexanderAmelkin — APPROVED** [review]

File: commands/bmc.vegman (hunk 2 above)
Comment (artemsen) [reply commands/bmc.vegman:236]: I mean, lets have the one only official way to do it.

Comment (nest1ing) [reply commands/bmc.vegman:236]: I think that `bmc syslog set` is shorter than `bmc syslog reset` ;)
And both ways work now.

Comment (artemsen) [reply commands/bmc.vegman:236]: So, now you have to describe both ways in the Users' documentation ;-)

**Review by artemsen — APPROVED** [review]

**Review by alatarum — APPROVED** [review]

**Pull request merged**